FaunaDB.Client/Client/DefaultClientIO.cs

```diff
@@ -34,6 +34,22 @@ class DefaultClientIO : IClientIO
 
         internal DefaultClientIO(HttpClient client, AuthenticationHeaderValue authHeader, LastSeen lastSeen, Uri endpoint, TimeSpan? timeout, Version httpVersion)
         {
+            if (client == null)
+            {
+                throw new ArgumentNullException(nameof(client));
+            }
+            if (authHeader == null)
+            {
+                throw new ArgumentNullException(nameof(authHeader));
+            }
+            if (endpoint == null)
+            {
+                throw new ArgumentNullException(nameof(endpoint));
+            }
+            if (lastSeen == null)
+            {
+                throw new ArgumentNullException(nameof(lastSeen));
+            }
```

Comment:
can we create an auxiliary method so we can "compact" those calls, something like:
```          
            AssertNotNull( client );
            AssertNotNull( authHeader );
            AssertNotNull( endpoint );
            AssertNotNull( lastSeen );
```

Reply:
Really it's good idea. Will do